common/djangoapps/config_models/admin.py

```diff
@@ -55,19 +64,21 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
     def save_model(self, request, obj, form, change):
         obj.changed_by = request.user
         super(ConfigurationModelAdmin, self).save_model(request, obj, form, change)
+        cache.delete(obj.cache_key_name(*(getattr(obj, key_name) for key_name in obj.KEY_FIELDS)))
+        cache.delete(obj.key_values_cache_key_name())
```

Comment:
Why aren't these cache deletions happening as part of the model actually being saved?